internal/kafka/internal/handlers/validation.go

```diff
@@ -24,6 +25,35 @@ var ValidKafkaClusterNameRegexp = regexp.MustCompile(`^[a-z]([-a-z0-9]*[a-z0-9])
 
 var MaxKafkaNameLength = 32
 
+func ValidateBillingCloudAccountIdAndMarketplace(ctx context.Context, kafkaService *services.KafkaService, kafkaRequestPayload *public.KafkaRequestPayload) handlers.Validate {
```

Comment:
Not necessary a blocker for this PR as it can be done in a separate JIRA but it'll be good to have a unit test for this

Reply:
Agree, part of this is covered here: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1075/files#diff-41495e01635585b99ecf9d0492c26b124d499699d317246c73b4f18e255a982dR51